requirements.txt

```diff
@@ -1,5 +1,7 @@
+amplitf
 numpy
 progress
 pyyaml
+sympy
```

Comment:
Why is that a requirement? Should be included via amplitf right?

Reply:
I saw it is used in `amplitude.py`

Reply:
Oh that was just there because some functions of amplitf are copied in the current master. Once this is replaced with the correct amplitude building code later on, this requirement will disappear.

Reply:
So you can delete that

Reply:
Ok https://github.com/ComPWA/tensorwaves/pull/36/commits/8b1214cde9166b82db82e7f0f3426d853857fa22